core/keyboard_nav/key_map.js

```diff
@@ -78,7 +78,13 @@ Blockly.user.keyMap.setKeyMap = function(keyMap) {
  * @package
  */
 Blockly.user.keyMap.getKeyMap = function() {
-  return Object.assign({}, Blockly.user.keyMap.map_);
+  var keyMap = Blockly.user.keyMap.map_;
```

Comment:
Can we instead create a polyfill for this in Blockly.utils.object.

Reply:
There already is: Blockly.utils.object.mixin